plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -93,6 +98,16 @@ public AzureBlobStore(RepositoryMetadata metadata, AzureStorageService service,
         final Map<String, AzureStorageSettings> prevSettings = this.service.refreshAndClearCache(emptyMap());
         final Map<String, AzureStorageSettings> newSettings = AzureStorageSettings.overrideLocationMode(prevSettings, this.locationMode);
         this.service.refreshAndClearCache(newSettings);
+        this.getMetricsCollector = (requestMethod) -> {
+            // BlobInputStream request information about the blob and those requests
+            // are not considered GET requests.
+            if (requestMethod.equalsIgnoreCase("HEAD")) {
```

Comment:
These are https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob-properties requests right? I'm not sure about the cost impact of these (would have to research that in more detail) but I would suggest we just track them as a separate counter for now and investigate that later instead of dropping them?